cmd/mysterium_client/command_run/command.go

```diff
@@ -75,3 +79,23 @@ func (cmd *CommandRun) Kill() {
 	cmd.communicationClient.Stop()
 	cmd.vpnClient.Stop()
 }
+
+func getVpnSession(sender communication.Sender, proposalId string) (session vpn_session.VpnSession, err error) {
+	sessionJson, err := sender.Request(communication.SESSION_CREATE, proposalId)
+	if err != nil {
+		return
+	}
+
+	var response vpn_session.SessionCreateResponse
+
+	err = json.Unmarshal([]byte(sessionJson), &response)
+	if err != nil {
+		return
+	}
+
+	if response.Success == false {
+		return session, errors.New(response.Message)
```

Comment:
Error might return empty. I suggest adding prefix string

Reply:
Don't see how that could happen. All edge cases are handled. What did I miss?:)

Reply:
Because it's external message, can come from another node's code